engine/runtime/src/main/java/org/enso/interpreter/runtime/error/PanicException.java

```diff
@@ -153,6 +158,7 @@ static Object handleExceptionMessage(
     try {
       return Text.create(strings.asString(text));
     } catch (UnsupportedMessageException e) {
+      ctx.getLogger().log(Level.WARNING, "Cannot convert " + text + " to string", e);
```

Comment:
@hubertp, is this the right way to dump additional info into CI log including:
- a message
- value of `text`
- exception `e` with its stacktrace

If not, can you please update the line to dump such an info so it appears in the CI log when the error happens?